pombola/south_africa/templates/core/person_detail.html

```diff
@@ -316,7 +316,7 @@ <h3>Party</h3>
         {% endfor %}
       </ul>
 
-    {% if person.everypolitician_uuid and email_contacts %}
+    {% if email_contacts %}
```

Comment:
I think we should replace everypolitician_uuid with has_ever_been_member_of_national_assembly